Source/WebCore/platform/network/NetworkStorageSession.cpp

```diff
@@ -35,13 +35,21 @@
 #include "RuntimeApplicationChecks.h"
 #include <wtf/NeverDestroyed.h>
 #include <wtf/ProcessPrivilege.h>
+#include <wtf/RunLoop.h>
 
 #if ENABLE(PUBLIC_SUFFIX_LIST)
 #include "PublicSuffix.h"
 #endif
 
 namespace WebCore {
 
+static HashSet<OrganizationStorageAccessPromptQuirk>& updatableStorageAccessPromptQuirks()
+{
+    ASSERT(RunLoop::isMain());
+    static MainThreadNeverDestroyed<HashSet<OrganizationStorageAccessPromptQuirk>> set;
```

Comment:
Can this be a member of the NetworkStorageSession rather than a process-global set?

Reply:
I was preferring a process-global set because it's we only need a single copy of it for the process.

Reply:
This should probably have a FIXME comment indicating our intent to move it to something owned by an instance of an object, probably NetworkStorageSession.